src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -265,6 +265,9 @@
     TEST_IGNORE_SIGAR("cassandra.test.ignore_sigar", "false"),
     PAXOS_EXECUTE_ON_SELF("cassandra.paxos.use_self_execution", "true"),
 
+    // properties for guardrails
```

Comment:
Changed to JavaDoc. I think that the other line comments around are being used to comment groups of related properties, instead of individual properties.